server/speech.js

```diff
@@ -0,0 +1,106 @@
+// This endpoint handles server calls to get transcript of an audio file
```

Comment:
@keving17 This is really well done with the way you've factored this into the different pieces! 👍